src/dhwcalc.cpp

```diff
@@ -24,6 +24,7 @@
 
 #include "cnguts.h"
 #include "exman.h"
+#include "hvac.h"
```

Comment:
Why is this needed? Please add comment to indicate its use.

Reply:
Existing definitions for derived courier class were in "hvac.h". (Related data structures are mentioned in the PR description.)

Reply:
Please add this comment to the file (not just on GitHub). i.e.:

```c++
#include "hvac.h" // CourierMsgHandler, CourierMsgHandlerBase, MessageLevel
```

Reply:
Done.